docs/content/kv_commands.md

```diff
@@ -7,19 +7,75 @@ The `kv` subcommand allows you to store application data in the Cloudflare netwo
 * have a Wrangler project set up with your `account_id` configured in the `wrangler.toml`
 * call commands from within a Wrangler project directory.
 
+## `kv:namespace`
+
+### `create`
+
+Creates a new namespace.
+
+#### Usage
+
+```console
+$ wrangler kv:namespace create "new kv namespace"
+🌀  Creating namespace with title "worker-new kv namespace"
+✨  Success: WorkersKvNamespace {
+    id: "e29b263ab50e42ce9b637fa8370175e8",
+    title: "worker-new kv namespace",
+}
+✨  Add the following to your wrangler.toml:
+kv-namespaces = [
+         { binding: "new kv namespace", id: "e29b263ab50e42ce9b637fa8370175e8" }
+]
+```
+
+### `list`
+
+Outputs a list of all KV namespaces associated with your account id.
+
+#### Usage
+
+```console
+$ wrangler kv:namespace list
+🌀  Fetching namespaces...
+✨  Success:
++---------------+----------------------------------+
+| TITLE         | ID                               |
++---------------+----------------------------------+
+| New Namespace | f7b02e7fc70443149ac906dd81ec1791 |
++---------------+----------------------------------+
+```
+
 Most `kv` commands require you to specify a namespace. A namespace can be specified in two ways:
-1. With a `--binding`: 
+
+1. With a `--binding`:
     ```sh
-    $ wrangler kv:key get --binding=KV "my key"
+    wrangler kv:key get --binding=KV "my key"
     ```
 1. With a `--namespace_id`:
     ```sh
-    $ wrangler kv:key get --namespace-id=06779da6940b431db6e566b4846d64db "my key"
+    wrangler kv:key get --namespace-id=06779da6940b431db6e566b4846d64db "my key"
     ```
 
-Most `kv` subcommands allow you to specify an optional `--env`, or environment, variable. This allows you to gate
-distinct namespaces within their own environments. For example, you could use seperate staging and production
-namespaces for KV data in your wrangler.toml:
+### `delete`
+
+Deletes a given namespace.
+
+Requires `--binding` or `--namespace-id` argument.
+
+Takes an optional `--env` [environment](./environments.md) argument.
+
+#### Usage
+
+```console
+$ wrangler kv:namespace delete --binding=KV
+Are you sure you want to delete namespace f7b02e7fc70443149ac906dd81ec1791? [y/n]
+yes
+🌀  Deleting namespace f7b02e7fc70443149ac906dd81ec1791
+✨  Success
+```
+
+Most `kv` subcommands also allow you to specify an environment with the optional `--env` flag. This allows you to publish workers running the same code but with different namespaces. For example, you could use separate staging and production namespaces for KV data in your `wrangler.toml`:
```

Comment:
Similarly, this --env documentation is completely buried... i think it's still worth putting up front.